v3/util/ca.go

```diff
@@ -62,3 +62,15 @@ func IsServerAuthCert(cert *x509.Certificate) bool {
 	}
 	return false
 }
+
+func IsEmailProtectionCert(cert *x509.Certificate) bool {
+	if len(cert.ExtKeyUsage) == 0 {
```

Comment:
I can try.  I'm sure I can come up with something better than "Because the function above for IsServerAuthCert does it this way."  I guess the main reason to return true where there are no EKUS is that this approach errs on the side of applying linting but it also seems like a mistake as the BRs apply where a certificate policy is present so this seems a backwards.  If anything we should have lints that enforce EKUs based on certificate policies rather than applying rules based on EKU...

Reply:
Policy wise, I think that makes sense, however, I think in the ServerAuth case it's more likely because ServerAuth linting is the "default" linting behavior of zlint. In other words, due tot he way that zlint is user pre-issuance, it may be possible that a linted cert doesn't have its EKUs yet or something (it seems wrong to me, but also seems like something that could happen).

Reply:
I've taken a swing at writing a comment for this function to explain the reasoning.  Let me know if you have any comments/complaints/suggestions